torch/_dynamo/backends/distributed.py

```diff
@@ -530,21 +550,26 @@ def compile_fn(self, gm: fx.GraphModule, example_inputs: List[torch.Tensor]):
                     try:
                         target_mod = gm.get_submodule(node.args[0].target)
                     except AttributeError:
+                        # This handles situations like  tmp = torch.mm(x, self.weight.t())
+                        # t: "f32[512, 512]" = l_self_seq_2_weight.t();  l_self_seq_2_weight = None
+                        # tmp: "f32[512, 512]" = torch.mm(input_2, t);  input_2 = t = None
+
+                        # DELETE THIS COMMENT BEFORE LANDING SHALL WE ONLY HANDLE .t()
+                        self.add_param_args(buckets[0], node)
```

Comment:
Instead of keeping it here, you should just add this line in L564. 

This way, we will support both cases - 1) existing case - where `node.args[0].target` is a module, and 2) new case - where it is an operation like `.t()` and we will just go through all the args (similar to call_function).

Does that make sense @wconstab ? I am not sure when is the case when both case 1 and case2 are True.

Reply:
i dont know if i followed your suggestion @anijain2305 but maybe if i see the code change it'll make sense

Reply:
@wconstab code updated